packages/webviz-core/src/dataProviders/BagDataProvider.js

```diff
@@ -129,7 +128,7 @@ export default class BagDataProvider implements DataProvider {
       await remoteReader.open(); // Important that we call this first, because it might throw an error if the file can't be read.
       if (remoteReader.size() === 0) {
         sendNotification("Cannot play invalid bag", "Bag is 0 bytes in size.", "user", "error");
-        return new Promise(() => {}); // Just never finish initializing.
+        return new Promise(() => { }); // Just never finish initializing.
```

Comment:
What happens from the user perspective when we do this? Do they see the sort of "empty" webviz state and received the notification message pop up in the UI?

Reply:
Yeah top right they get a red X with that message